Fundamentals/first_letter.py

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2016 Ami . All rights reserved.
+
+
+def user_input():
```

Comment:
`user_input` doesn't describe well what this function actually does. Maybe `get_user_password_and_print_first_letter`. Or you can separate the 2 operations: one function for getting the user password and one function to print its first letter, and name them accordingly.